core/inject.js

```diff
@@ -155,10 +155,12 @@ Blockly.createMainWorkspace_ = function(svg, options, blockDragSurface,
   svg.appendChild(mainWorkspace.createDom('blocklyMainBackground'));
 
   // Set the theme name and renderer name onto the injection div.
+  var rendererName = mainWorkspace.getRenderer().name;
+  var themeName = mainWorkspace.getTheme().name;
   Blockly.utils.dom.addClass(mainWorkspace.getInjectionDiv(),
-      (wsOptions.renderer || 'geras') + '-renderer');
+      rendererName + '-' + themeName + '-renderer');
```

Comment:
With this change, do we need to add additional logic to ThemeManager.setTheme to update class combo?

Reply:
Good point, I'll look into that.